lib/jobs.js

```diff
@@ -44,6 +44,13 @@ async function unzip(config) {
         throw err;
     }
 
+    try {
+        await store.deleteZip(config.buildId, config.token);
+    } catch (err) {
+        logger.error(err.message);
+        throw err;
```

Comment:
if deleting zip is failed, the whole process including above L29-L45 will be retried by resque retry plugin?

Reply:
After testing in local, I checked that the behavior is to retry only the deletion of the zip file.

Reply:
Remove throw to prevent retrying unzip on delete failure.
The delete retry is performed by `got`, so there should be no problem.